linera-rpc/src/mass_client.rs

```diff
@@ -6,13 +6,13 @@ use async_trait::async_trait;
 use thiserror::Error;
 
 #[derive(Error, Debug)]
-pub enum MassClientError {
+pub enum Error {
```

Comment:
Why do this rename?
Over the code, we use Error which have a prefix. Why this loss of information?

Reply:
The information is already contained in the module name `mass_client`, so `mass_client::Error` is sufficient — there's no need to duplicate it as `mass_client::MassClientError`.

As a reductio ad absurdum, equivalently one could ask why the error isn't named `LineraProtocolLineraRpcMassClientError` :)